packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AuthCognito.kt

```diff
@@ -182,7 +182,7 @@ public class AuthCognito : FlutterPlugin, ActivityAware, MethodCallHandler, Plug
       "signOut" ->  onSignOut(result)
       "updatePassword" -> onUpdatePassword(result, data)
       "resetPassword" -> onResetPassword(result, data)
-      "confirmPassword" -> onConfirmPassword(result, data)
+      "confirmResetPassword" -> onConfirmResetPassword(result, data)
```

Comment:
Note: This PR deprecates confirmPassword() and creates a new confirmResetPassword() to match the native APIs. The swift/kotlin code was updated as well as part of this, which resulted in a fair amount of changes.